### PR TITLE
Remove Stormpath

### PR DESCRIPTION
**Pull request by DwayneJengSage**
This code change completes the Stormpath migration. It removes all Stormpath code and most references to Stormpath from our BridgePF codebase.

Testing done:
* all unit tests and integ tests
* manual tests that aren't covered by automation, such as email verification and reset password workflows
* manually tested basic account scenarios, such as sign up, sign in, and user profile

### CODE REVIEW

File: app/org/sagebionetworks/bridge/stormpath/StormpathAccount.java [hunk 1]
```diff
@@ -1,336 +0,0 @@
-package org.sagebionetworks.bridge.stormpath;
```
Comment (alxdarksage) [inline app/org/sagebionetworks/bridge/stormpath/StormpathAccount.java:1]: Goodbye StormpathAccount, I will not miss you!

**Review by alxdarksage — APPROVED** [review]
Looks good.

File: app/org/sagebionetworks/bridge/config/BridgeConfig.java [hunk 2]
```diff
@@ -15,10 +15,6 @@
     private static final String TEMPLATE_CONFIG = CONFIG_FILE;
     private static final String LOCAL_CONFIG = System.getProperty("user.home") + "/" + ".bridge" + "/" + CONFIG_FILE;
 
-    private static final String ENTERPRISE_STORMPATH_ID = "enterprise.stormpath.id";
-    private static final String ENTERPRISE_STORMPATH_SECRET = "enterprise.stormpath.secret";
-    private static final String ENTERPRISE_STORMPATH_APPLICATION_HREF = "enterprise.stormpath.application.href";
```
Comment (zaro0508) [inline app/org/sagebionetworks/bridge/config/BridgeConfig.java:20]: Should these VARs be removed from the bridge-server.conf[1]  and heroku/AWS ENV config?

[1] https://github.com/Sage-Bionetworks/BridgePF/blob/develop/conf/bridge-server.conf#L44

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: app/org/sagebionetworks/bridge/config/BridgeConfig.java (hunk 2 above)
Comment (DwayneJengSage) [reply app/org/sagebionetworks/bridge/config/BridgeConfig.java:20]: They have already been removed from bridge-server.conf. We can remove them from heroku/AWS after this change is pushed out.

**Pull request merged**